client/blocks/login/index.jsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import LoginForm from './login-form';
+import TwoFactorAuthentication from './two-factor-authentication';
+import { isTwoFactorEnabled } from 'state/login/selectors';
+
+const Login = ( { twoFactorEnabled, translate } ) => {
+	return twoFactorEnabled
+		? ( <TwoFactorAuthentication /> )
+		: ( <LoginForm
+			title={ translate( 'Sign in to connect to WordPress.com' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Sign in to WordPress.com' )` *ES Score: 9.42*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/login-component)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).